docs/reference/api/security/TLS.md

```diff
@@ -41,6 +41,26 @@ Then create a file named `mbed_app.json` at the root of your application with th
 }
 ```
 
+### Mbed TLS platform context
+
+Some hardware accelerators require initialization, regardless of the specific cryptography engine. For this, we introduced `mbedtls_platform_setup()` and `mbedtls_platform_terminate()`.
+
+As the [examples](#mbed-tls-examples) show, you *must* call the `mbedtls_platform_setup()` function before you call any Mbed TLS API. After using the Mbed TLS API, you *must* call `mbedtls_platform_teardown()` to terminate the platform hardware driver. We suggest you set the platform context parameter for these functions as `NULL`.
+
+```
+    int exit_code = MBEDTLS_EXIT_FAILURE;
```

Comment:
Since this is an Mbed OS related example, I am not sure, using `MBEDTLS_EXIT_FAILURE` and such is suitable.

Reply:
ok. Note that this is still an Mbed TLS documentation, on the Mbed OS.